app/models/SupportFrontendSwitches.scala

```diff
@@ -3,56 +3,26 @@ package models
 import io.circe.generic.extras.Configuration
 import io.circe.generic.extras.auto._
 import io.circe.{Decoder, Encoder}
-import utils.Circe.decodeStringAndCollect
+import enumeratum.{CirceEnum, Enum, EnumEntry}
 
-/**
-  * Based on https://github.com/guardian/support-frontend/blob/master/support-frontend/app/admin/settings/
-  */
+import scala.collection.immutable.IndexedSeq
 
-sealed trait SwitchState
-case object On extends SwitchState
-case object Off extends SwitchState
+sealed trait SwitchState extends EnumEntry
```

Comment:
this bit is just (slightly) simplifying the `SwitchState` json support by using enumeratum

Reply:
it's a nice idea but I'm not sure it's a big saving - I guess it just maps On to "On" and Off to "Off" transparently?

Reply:
I've just discovered circe now has a nicer way to do this, I've pushed up a commit